pyomo/gdp/tests/common_tests.py

```diff
@@ -1438,28 +1438,6 @@ def check_deactivated_disjunct_leaves_nested_disjunct_active(self,
     self.assertTrue(m.d1.d4.active)
     self.assertFalse(m.d1.d4.indicator_var.fixed)
 
-def check_mappings_between_disjunctions_and_xors(self, transformation):
```

Comment:
Is this test irrelevant now? Should we not test the mappings now that there is a unified interface for the transformations?

Reply:
Yeah, I think this is the one you answered below. It's just way simpler. I don't want the test to rely on how I name things.